src/test/functional/utlis/puppeteer.util.ts

```diff
@@ -61,6 +61,15 @@ export const getElements = async (selector: string) => {
   }
 };
 
+export const getHtmlFromElements = async (el: string) => {
+  try {
+    return await scope.page.$$eval(el, (elements: any) => elements.map((e: any) => e.innerHTML));
+  } catch (error) {
+    console.log("The element didn't appear.");
```

Comment:
I think we should log the element not found here, similar to other methods in this class.

Reply:
Helps when debugging failing tests.